c2corg_common/fields_book.py

```diff
@@ -27,6 +28,7 @@
     'author',
     'quality',
     'publication_date',
+    'langs',
```

Comment:
Ok, I deleted other fields and left only these with book_types, which is useful to see when filtering and some books cover more topics.